src/neuroconv/__init__.py

```diff
@@ -1,4 +1,11 @@
 from .nwbconverter import NWBConverter
-from .datainterfaces import *
 from .tools import spikeinterface, roiextractors, neo
 from .tools.yaml_conversion_specification import run_conversion_from_yaml
+
+
+def __getattr__(name: str):
+    from importlib import import_module
```

Comment:
why have this import here?

Reply:
If we import in the global space it would be included in the `__dir__` so you could do something weird like `from neuroconv import importlib`. I guess we could overwrite that however. IDK feels safer to do it locally to where it's needed